### ISSUE TITLE
var declaration problems

### ISSUE THREAD

**Issue by withIssues** [lukehaas/RunJS#518]
var declarations work properly only under strict mode but under strict mode there is no error report.

```javascript
//"use strict";

var b = 2, a = 1; // var declaration problems, ok with let and const
setTimeout(function(){
  console.log(a); // undefined under non strict mode
});

a(); // no error report under strict mode

console.log('ok');
```

**Comment by lukehaas** [comment]
Thanks, this will be fixed in the next release.

**Comment by lukehaas** [comment]
This is now resolved in the latest release - version 2.7.5
